src/main/java/mate/academy/model/Movie.java

```diff
@@ -0,0 +1,49 @@
+package mate.academy.model;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "movies")
+public class Movie {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    private String title;
+    private String description;
+
```

Comment:
let's explicitly create a public empty constructor

Reply:
But in checklist it was said not to create redundant default constructor

Reply:
sorry, it's my mistake 😢 
I has written about it in the slack channel (in thread)